runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningRocksOverlay.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2019, Adam <Adam@sigterm.info>
```

Comment:
Copyright name and date

Reply:
I'm pretty sure this one is actually correct.

Reply:
Follow the commit history. This is Adam's work. I renamed the class.